Kentico.Kontent.Management/IManagementClient.cs

```diff
@@ -239,11 +239,11 @@ public interface IManagementClient
         /// <summary>
         /// Returns strongly typed listing of assets.
```

Comment:
Is that true? I briefly went through the [implementation](https://github.com/Kentico/kontent-management-sdk-net/blob/vNext/Kentico.Kontent.Management/ManagementClient.cs#L769-L779) and it looks it is doing paging automatically.